src/main/java/Duke.java

```diff
@@ -1,10 +1,135 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    static final int ITEMS = 25;
+    static final int LINES = 85;
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
+
         System.out.println("Hello from\n" + logo);
+
+        for (int i = 0; i < LINES; i++) {
```

Comment:
Perharps you can just create the string for the horizontal line using a method instead of a for loop to print our _ 25 times

Reply:
agree, name it `printIntroText()`